pkgs/development/compilers/cudatoolkit/hooks/setup-cuda-hook.sh

```diff
@@ -55,8 +58,9 @@ setupCUDAToolkitCompilers() {
 
     # CMake's enable_language(CUDA) runs a compiler test and it doesn't account for
     # CUDAToolkit_ROOT. We have to help it locate libcudart
-    if [[ -z "${nvccDontPrependCudartFlags-}" ]] ; then
-        export NVCC_APPEND_FLAGS+=" -L@cudartLib@/lib -L@cudartStatic@/lib -I@cudartInclude@/include"
+    local cudartFlags="@cudartFlags@"
+    if [[ -z "${nvccDontPrependCudartFlags-}" ]] && [[ -n "\${cudartFlags:-}" ]] ; then
```

Comment:
How is `"\${cudartFlags:-}"` interpreted (if it is) given `$` is escaped? I would think that this test always evaluates to false because there's no string interpolation happening and it's not empty.

Reply:
Holy mother, I must've copied that from a nix expression